readthedocs/projects/migrations/0027_remove_unused_field.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2018-07-28 02:46
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0026_ad-free-option'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='project',
+            name='allow_comments',
+        ),
+        migrations.RemoveField(
+            model_name='project',
+            name='comment_moderation',
+        ),
+        migrations.AlterField(
```

Comment:
Why this Altering?

Reply:
The rest of alters are from missing migrations (just changes in help text or options) https://github.com/rtfd/readthedocs.org/issues/4201